superset/views/base.py

```diff
@@ -254,6 +254,19 @@ def json_response(
             mimetype="application/json",
         )
 
+    def render_app_template(self) -> FlaskResponse:
+        payload = {
+            "user": bootstrap_user_data(g.user),
+            "common": common_bootstrap_payload(),
+        }
+        return self.render_template(
+            "superset/crud_views.html",
```

Comment:
should the base superset view always return the crud_views template? That seems a bit odd to me.

Reply:
As state below this is repeating another pattern found in other list views. This method is cloned over from `SupersetModelView` (https://github.com/preset-io/incubator-superset/blob/690689c5caed2becf3a1b6a322fb882df082022a/superset/views/base.py#L338) since inheriting from `SupersetModelView` in the `Superset` class was causing an error.